### PR TITLE
Complete Reset password APIs

### PR DESCRIPTION
**Pull request by harmeshverma01**

### CODE REVIEW

File: user/views.py [hunk 1]
```diff
@@ -132,18 +131,19 @@ def post(self, request):
...
-        confirm_password = request.data.get('confirm_password')
-        user = User.objects.filter(email=email)
-        if user.check_password(email):
-            password.validate_password(password=request.data, user=User)
-            return password
-        
-        return Response(({'details', 'create a password'}))
-        
-        
+        user = User.objects.get(email=email)
```
Comment (nodeexcel) [inline user/views.py:140]: Why we are not using try except or get_object_or_404 here ?

File: user/views.py [hunk 2]
```diff
@@ -132,18 +131,19 @@ def post(self, request):
...
+        if user.check_password(password):
```
Comment (nodeexcel) [inline user/views.py:141]: add validation properly for required request parameters

File: user/views.py [hunk 3]
```diff
@@ -132,18 +131,19 @@ def post(self, request):
...
+            serializer = ResetpasswordSerializer(data=request.data)
+            if serializer.is_valid():
+                return Response(({'message': 'password changed successfully'}), status=status.HTTP_200_OK)
```
Comment (nodeexcel) [inline user/views.py:144]: totally wrong

File: user/views.py (hunk 1 above)
Comment (harmeshverma01) [reply user/views.py:140]: fixed

File: user/views.py (hunk 2 above)
Comment (harmeshverma01) [reply user/views.py:141]: fixed

File: user/views.py (hunk 3 above)
Comment (harmeshverma01) [reply user/views.py:144]: fixed

**Pull request merged**